README.md

```diff
@@ -88,6 +88,7 @@ Grammars:
 - [TON Jest](https://github.com/tact-lang/ton-jest): Testing tools for TON and Jest.
 - [Tact Deployer](https://github.com/tact-lang/tact-deployer): A library to prepare the deployment of a Tact package.
 - [Foton](https://foton.sh): A TypeScript toolkit for interacting with TON wallets and blockchain as a whole.
+- [Tonion](https://github.com/ton-ion/tonion-contracts): A collection of secure and reusable smart contract Traits for the Tact language on the TON blockchain.
```

Comment:
Thanks! The description says "secure" but there is no link to a security audit report. If we are to endorse your project, we need some evidence of security

Reply:
Hi @anton-trunov 
Maybe we should remove the "secure" adjective since we haven't undergone a security audit yet. We have just started, but it is part of our roadmap for the future after we achieve stability and completeness.

Reply:
Sounds good. And we can always revise this in the future

Reply:
thanks, @anton-trunov 
I changed the description

Reply:
Hmmm, when someone clicks the link, they go to your GitHub repo, which currently states that it's secure. Can you also please fix that?

Reply:
fixed